### PR TITLE
IGVF-1507-integrated-in

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

**Review by gabdank — DISMISSED** [review]

File: src/igvfd/schemas/changelogs/tabular_file.md [hunk 1]
```diff
@@ -20,6 +21,7 @@
 * Add `barcode to sample mapping` to `content_type`.
 * Add `release_timestamp`.
 * Add `MPRAbase` to `collections`.
+* Add `integrated_in`.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/tabular_file.md:24]: why do you report add it here, but then again that it was added as calculated?

**Review by gabdank — CHANGES_REQUESTED** [review]
minor changelog and description comments

File: src/igvfd/types/file.py [hunk 2]
```diff
@@ -103,12 +103,29 @@ class File(Item):
...
+    }
 
     set_status_up = [
         'file_format_specifications'
     ]
     set_status_down = []
 
+    @calculated_property(schema={
+        'title': 'Integrated In',
+        'description': 'Construct library sets that this file was used for in insert design or directly cloned as a vector.',
```
[the remark this answers is not in the document]
Comment (ian-whaling) [reply src/igvfd/types/file.py:117]: done

File: src/igvfd/schemas/changelogs/tabular_file.md (hunk 1 above)
Comment (ian-whaling) [reply src/igvfd/schemas/changelogs/tabular_file.md]: fixed

File: src/igvfd/schemas/changelogs/reference_file.md [hunk 3]
```diff
@@ -13,6 +14,7 @@
 * Add `MPRAbase` to `collections`.
 * Add `proteins_proteins`, `regulatory_regions_genes`, `regulatory_regions_genes_biosamples`, `regulatory_regions_genes_biosamples_donors`, `regulatory_regions_genes_biosamples_treatments_chebi`, `regulatory_regions_genes_biosamples_treatments_proteins`, and `variants_genes_terms` to `content_type`.
 * Restrict submission of `guide RNA sequences`, `sequence barcodes`, `vector sequences` to admin users.
+* Add `integrated_in`.
```
[the remark this answers is not in the document]
Comment (ian-whaling) [reply src/igvfd/schemas/changelogs/reference_file.md]: fixed

**Review by gabdank — DISMISSED** [review]

**Pull request merged**